src/geom/point.jl

```diff
@@ -47,34 +47,45 @@ function render(geom::PointGeometry, theme::Gadfly.Theme, aes::Gadfly.Aesthetics
     Gadfly.assert_aesthetics_equal_length("Geom.point", aes, :x, :y)
 
     default_aes = Gadfly.Aesthetics()
-    default_aes.shape = Function[Shape.circle]
-    default_aes.color = discretize_make_ia(RGBA{Float32}[theme.default_color])
+    default_aes.shape = Function[theme.point_shapes[1]]
+    default_aes.color = RGBA{Float32}[theme.default_color]
     default_aes.size = Measure[theme.point_size]
-    default_aes.alpha = [theme.alphas[1]]
+    default_aes.alpha = Float64[theme.alphas[1]]
     aes = inherit(aes, default_aes)
 
-    if eltype(aes.size) <: Int
+    aes_size = if eltype(aes.size) <: Int
       size_min, size_max = extrema(aes.size)
       size_range = size_max - size_min
       point_size_range = theme.point_size_max - theme.point_size_min
-      interpolate_size(x) = theme.point_size_min + (x-size_min) / size_range * point_size_range
```

Comment:
What was even the point of creating this function originally?

Reply:
`Scale.size_continuous` should handle the mapping from the `size` aesthetic values  to 0-1 (proportion), as for the e.g. `alpha` aesthetic, so the scaling for `size` wasn't properly implemented in grammar of graphics style (unfortunately).  This would be good to implement when I do `Guide.sizekey`.